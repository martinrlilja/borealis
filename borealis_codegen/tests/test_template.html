<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Test</title>
    </head>
    <body>
        <article>
            <h1>Test Document</h1>
            <p>This is a very fancy test document.</p>

            <img src="img.png" alt="and image">

            <p>Another paragraph.</p>
        </article>
    </body>
</html>
