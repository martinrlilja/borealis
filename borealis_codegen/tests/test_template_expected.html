<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Test</title>
    </head>
    <body>
        <article>
            <h1>Test Document</h1>
            <p>This is a very fancy test document.</p>

            <img src="img.png" alt="an image">

            <p>Another "paragraph".}}</p>

            <div>a counter: 4</div>

            <h2>Test template says: <span>Test</span></h2>

            <p>1, 2, 3, 4, </p>
        </article>
    </body>
</html>
